policyengine_canada/parameters/gov/provinces/yt/benefits/rebates/ygcpri/amount/child.yaml

```diff
@@ -0,0 +1,9 @@
+description: Yukon provides this base amount for each child under the Yukon Government carbon price rebate (YGCPRI).
+values:
+  2023-01-01: 320
+metadata:
+  unit: currency-CAD
+  label: Yukon child YGCPRI base
+  reference:
+  - title: Yukon Government carbon price rebate - Individuals (YGCPRI)
```

Comment:
Add a legal reference, or comment that it's unavailable if you can't find it.

The law may be more parsimonious than this document, e.g. one amount and 10% for non-Whitehorse.